tailwind.config.js

```diff
@@ -0,0 +1,46 @@
+
+module.exports = {
+  purge: {
+    mode: 'production',
+    content: ['./src/**/*.html'],
+  },
+  theme: {
+    extend: {},
+    colors: {
```

Comment:
Isn't this a perfect spot to add the board color?

Reply:
I guess it is, but then we could also not include a whole lot of the other colors and use a consistent color scheme instead. That feels a bit beyond the scope of this PR though.